app/models/split.rb

```diff
@@ -80,11 +80,11 @@ def name_must_be_snake_case
   end
 
   def name_must_not_include_new
-    errors[:name] << "'new' is too vague. Use absolute time if needed e.g. 'summary_redesign_late_2015'" if name_contains_new?
+    errors[:name] << "should not contain the ambiguous word 'new'. If expressing timing, refer to absolute time like 'late_2015'. If expressing creation use 'create'." if name_contains_new?
   end
 
   def name_must_not_end_with_test
-    errors[:name] << "'test' is redundant.  All splits are testable." if name_ends_with_test?
+    errors[:name] << "should not end with 'test', as it is redundant. All splits are testable." if name_ends_with_test?
```

Comment:
This is a little funky come to think of it now that we recommend experiment as a suffix for testable splits but we can tighten that up later, presumably with client libs that only send _experiment and _enabled as split names so this basically never gets exercised anymore.